app/kuma-dp/cmd/run.go

```diff
@@ -54,5 +63,42 @@ func newRunCmd() *cobra.Command {
 			return nil
 		},
 	}
+
+	cmd.PersistentFlags().StringVarP(&args.name, "name", "", "", "Name of the Dataplane")
+	cmd.PersistentFlags().Uint32VarP(&args.adminPort, "admin-port", "", 0, "Port for Envoy Admin")
```

Comment:
Why we are not using actual default values here ?

Reply:
If we define defaults then setting config via env var would not work. If we want to drop it, we may just as well resign from the `config.Load()`.